package.json

```diff
@@ -40,7 +41,7 @@
             "eslint:recommended"
         ],
         "parserOptions": {
-            "parser": "babel-eslint"
+            "parser": "@babel/eslint-parser"
```

Comment:
Not sure if this is needed. Most places I saw that recommended this change had it in a different file. Additionally, I think there are other unchanged references to the old dependency